src/Persister/EntityPersister.php

```diff
@@ -143,17 +93,50 @@ public function refresh(int $id, object $entity)
 
     public function getDetachDeleteQuery($object): Statement
     {
-        $query = 'MATCH (n) WHERE id(n) = {id} DETACH DELETE n';
+        $query = sprintf("MATCH (n) WHERE id(n) = $this->paramStyle DETACH DELETE n", 'id');
```

Comment:
I don't understand what is the point of sprintf here?

Reply:
for switching between verisons